app/components/account/TokenAccountSection.tsx

```diff
@@ -497,3 +552,509 @@ function MultisigAccountCard({ account, info }: { account: Account; info: Multis
+                        <td className="text-lg-end">{extension.harvestToMintEnabled ? 'enabled' : 'disabled'}</td>
+                    </tr>
+                    <tr>
+                        <td>Withheld Amount</td>
+                        <td className="text-lg-end">{extension.withheldAmount}</td>
+                    </tr>
+                </>
+            );
+        }
+        case 'defaultAccountState': {
+            const extension = create(tokenExtension.state, DefaultAccountState);
+            return (
+                <tr>
+                    <td>DefaultAccountState</td>
+                    <td className="text-lg-end">{extension.accountState}</td>
+                </tr>
+            );
+        }
+        case 'nonTransferable': {
+            return (
+                <tr>
+                    <td>Non-Transferable</td>
+                    <td className="text-lg-end">enabled</td>
+                </tr>
+            );
+        }
+        case 'interestBearingConfig': {
+            const extension = create(tokenExtension.state, InterestBearingConfig);
+            return (
+                <>
+                    <tr>
+                        <h4>Interest-Bearing</h4>
+                    </tr>
+                    {extension.rateAuthority && (
+                        <tr>
+                            <td>Authority</td>
+                            <td className="text-lg-end">
+                                <Address pubkey={extension.rateAuthority} alignRight link />
+                            </td>
+                        </tr>
+                    )}
+                    <tr>
+                        <td>Current Rate Basis Points</td>
+                        <td className="text-lg-end">{extension.currentRate}</td>
+                    </tr>
+                    <tr>
+                        <td>Pre-Current Average Rate</td>
+                        <td className="text-lg-end">{extension.preUpdateAverageRate}</td>
+                    </tr>
+                    <tr>
+                        <td>Last Update Timestamp</td>
+                        <td className="text-lg-end">{extension.lastUpdateTimestamp}</td>
+                    </tr>
+                    <tr>
+                        <td>Initialization Timestamp</td>
+                        <td className="text-lg-end">{extension.initializationTimestamp}</td>
+                    </tr>
+                </>
+            );
+        }
+        case 'permanentDelegate': {
+            const extension = create(tokenExtension.state, PermanentDelegate);
+            if (extension.delegate) {
+                return (
+                    <tr>
+                        <td>Permanent Delegate</td>
+                        <td className="text-lg-end">
+                            <Address pubkey={extension.delegate} alignRight link />
+                        </td>
+                    </tr>
+                );
+            } else {
+                return <></>;
```

Comment:
If the permanent delegate is unset, it becomes impossible to ever set it again, similar to what we do with mint authority or freeze authority, so for consistency with everything else, I think it would be better to not show it.

CPI Guard is different because you can always toggle it